src/node/rpc/test/membervoting_test.cpp

```diff
@@ -372,7 +370,7 @@ struct NewMember
   Cert cert;
 };
 
-TEST_CASE("Add new members until there are 7, then reject")
+TEST_CASE("Add new members until there are 7")
```

Comment:
what happened to the `then reject` bit of this test?

Reply:
The long test name made it hard to select when running only one test case (`-tc=` doctest option). I'll re-add it.